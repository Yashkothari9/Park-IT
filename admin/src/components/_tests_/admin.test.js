import React from 'react';
import { shallow } from 'enzyme';
import MyManager from '../manager/manager';

test('should render User', () => {
    const wrapper = shallow(<MyManager/>);
    expect(wrapper).toMatchSnapshot();
});