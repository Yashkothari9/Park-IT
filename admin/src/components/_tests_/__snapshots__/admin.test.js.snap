// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render User 1`] = `ShallowWrapper {}`;
